code/core/package.json

```diff
@@ -254,7 +270,8 @@
     "express": "^4.19.2",
     "process": "^0.11.10",
     "recast": "^0.23.5",
-    "util": "^0.12.4",
```

Comment:
I removed this package, as it was a leftover from the reverted test consolidation attemt @ndelangen. Gets rid of 22 deps.

Reply:
It's used here:
https://github.com/storybookjs/storybook/blob/d2b58e0c2f2a58d80860fd2d926732eb49dc51d4/code/core/scripts/prep.ts#L126